### ISSUE TITLE
Unclear UX/Journey for some App users

### ISSUE THREAD

**Issue by FaGuoMa** [jitsi/jitsi-meet#6119]
**Is your feature request related to a problem you are facing?**
Users clicking on the links from our on-premise instance are (for a few) confused by the app landing page ("Download the app" or **Continue to the app**)

Not a huge deal, but

**Describe the solution you'd like**
I have edited the `/usr/share/jitsi-meet/libs/app.bundle.min.js` to "Continue to the **Meeting**" and it solved the issue to some extent (even though the lines "Download" and "Continue" might be better switched in position ("Continue" an "Download" bellow) in terms of natural flow ?

**Describe alternatives you've considered**
Maybe we could add *some* content to explain the difference.

**Comment by obfusk** [comment]
I also think that the wording could use some improvement. We currently have:

```
You need the Jitsi Meet mobile app to join this meeting on your phone.
[Download the app]
[Continue to the app]
```

I suggest something like:
```
You need the Jitsi Meet mobile app to join this meeting on your phone.
If you don't have the app yet: [Download the app].
If you already have the app: [Join this meeting using the app].
```

**Comment by FaGuoMa** [comment]
And maybe invert or make the CSS of both options the same?

**Comment by saghul** [comment]
@obfusk the suggestions sound good to me. Care to open a PR to discuss specifics?

**Comment by obfusk** [comment]
@saghul I just did :)

**Comment by obfusk** [comment]
@FaGuoMa I proposed a fix for this: #6717.
